src/modules/CurrencyModule.h

```diff
@@ -70,17 +70,17 @@ class CurrencyModule : public Module
 			return *this;
 		}
 
-		double currency() const
+        double currency() const
 		{
-			return m_Currency;
+            return (double)m_Currency / 100.0;
 		}
```

Comment:
I feel like this should also be changed to be an `int`. Basically changing everything relating to currency which is a `double` to use an `int` (number of cents).